# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xine-ui/xine-ui-0.99.6-no-update-mime-database.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Do not run 'update-mime-database' to avoid shared files

--- xine-ui-0.99.6/misc/Makefile.am.orig	2011-05-19 15:57:13.140002344 +0200
+++ xine-ui-0.99.6/misc/Makefile.am	2011-05-19 15:57:45.824002344 +0200
@@ -32,4 +32,3 @@
 	test -f xine-bugreport || $(LN_S) xine-check xine-bugreport
 
 install-data-hook:
-	test "x$(DESTDIR)" != "x" || update-mime-database $(mimedir)/..
